recipes/dirac/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "DIRAC" %}
+{% set version = "6.21.5" %}
+{% set major, minor, patch = version.split(".") %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
There are lots of packages with the name dirac. What's the name that other package managers use for this package?

Reply:
That's a good point. It's not in any other package managers but I've renamed it to `dirac-grid` which matches the website URL and GitHub organisation.